translation/es/patterns/service-vs-library.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+Servicio vs Biblioteca
```

Comment:
In general we should use "Librería" instead of "Biblioteca" if we are speaking about artefacts reused by different teams or projects (software libraries). "Biblioteca" is used more to describe the place to store books. I found that translation ("biblioteca" as library) along the patterns. 

Could we use just "Librería" for the same concept along this PR? Asking just to create multiple comments/reviews along the long list of files.

@ovas04 What do you think?